docs-src/references/dynamic-configuration.md

```diff
@@ -0,0 +1,209 @@
+| `matching.persistenceMaxQPS`              | Int  | Maximum queries per second that the Matching service host can query the Persistence store database.                                                                                                                                | 9000                    |
+| `matching.persistenceNamespaceMaxQPS`     | Int  | Maximum queries per second that the Matching host can query the Persistence store database for each Namespace. <br /> If the value set for this config is less or equal to 0, the value set for  `PersistenceMaxQS` will apply.    | 0                       |
+| Worker                                    |      |                                                                                                                                                                                                                                    |                         |
+| `worker.persistenceMaxQPS`                | Int  | Maximum queries per second that the Worker service host can query the Persistence store database.                                                                                                                                  | 100                     |
+| `worker.persistenceNamespaceMaxQPS`       | Int  | Maximum queries per second that the Worker host can query the Persistence store database for each Namespace. <br /> If the value set for this config is less or equal to 0, the value set for  `PersistenceMaxQS` will apply.      | 0                       |
+| Visibility                                |      |                                                                                                                                                                                                                                    |                         |
+| `system.visibilityPersistenceMaxReadQPS`  | Int  | Maximum queries per second that Visibility database be queried for read operations.                                                                                                                                                | 9000                    |
+| `system.visibilityPersistenceMaxWriteQPS` | Int  | Maximum queries per second that Visibility database be queried for write operations.                                                                                                                                               | 9000                    |
+
+#### Activity and Workflow default policy setting
+
+You can define default values for Activity and Workflow [Retry Policies](/concepts/what-is-a-retry-policy) at the Cluster level with the following dynamic configuration keys.
+Th, the default values will apply. Also, setting custom retry polici
```

Comment:
Something's missing in this line...

Reply:
thanks for catching this! Must've snuck in while I was juggling pages